actioncable/app/assets/javascripts/action_cable/connection.coffee

```diff
@@ -95,6 +95,8 @@ class ActionCable.Connection
           @subscriptions.notify(identifier, "connected")
         when message_types.rejection
           @subscriptions.reject(identifier)
+        when message_types.client_error
+          @subscriptions.notify(identifier, "error")
```

Comment:
What happens the server is updated and starts sending updates to the client without its knowledge? Will it be caught by the protocol stuff?